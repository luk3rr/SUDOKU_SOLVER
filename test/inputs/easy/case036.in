907108003 300700000 000064708 690005000 028000000 705086000 079000400 200041009 040007800 
