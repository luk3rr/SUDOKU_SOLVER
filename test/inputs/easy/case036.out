Solving the following grid:
9 0 7 | 1 0 8 | 0 0 3 
3 0 0 | 7 0 0 | 0 0 0 
0 0 0 | 0 6 4 | 7 0 8 
------+-------+------
6 9 0 | 0 0 5 | 0 0 0 
0 2 8 | 0 0 0 | 0 0 0 
7 0 5 | 0 8 6 | 0 0 0 
------+-------+------
0 7 9 | 0 0 0 | 4 0 0 
2 0 0 | 0 4 1 | 0 0 9 
0 4 0 | 0 0 7 | 8 0 0 


Solution found :')

9 6 7 | 1 5 8 | 2 4 3 
3 8 4 | 7 2 9 | 1 5 6 
5 1 2 | 3 6 4 | 7 9 8 
------+-------+------
6 9 1 | 2 7 5 | 3 8 4 
4 2 8 | 9 1 3 | 5 6 7 
7 3 5 | 4 8 6 | 9 2 1 
------+-------+------
8 7 9 | 6 3 2 | 4 1 5 
2 5 3 | 8 4 1 | 6 7 9 
1 4 6 | 5 9 7 | 8 3 2 

Algorithm: BFS
Total time: 81 ms
Total expanded states: 7872
